config/routes.rb

```diff
@@ -10,4 +10,8 @@
     )
   end
 
+  resources :errors, only: [:show]
+
```

Comment:
Is `resources :errors` needed here? I tried briefly uncommenting it and it still worked